indico/migrations/versions/201708231635_dc2c2fa6f5be_add_file_md5_checksum.py

```diff
@@ -0,0 +1,60 @@
+"""Add file MD5 checksum
+
+Revision ID: dc2c2fa6f5be
+Revises: ebca3e91b139
+Create Date: 2017-08-23 16:35:36.634285
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'dc2c2fa6f5be'
+down_revision = 'ebca3e91b139'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('files', sa.Column('md5', sa.String, nullable=False, server_default='', index=True),
```

Comment:
does this actually work? AFAIK `index=True` is a no-op if the table already exists (or at least i think it used to be the case)